website/server/libs/slack.js

```diff
@@ -77,14 +89,75 @@ function sendFlagNotification ({
   });
 }
 
+function sendInboxFlagNotification ({
+  authorEmail,
+  flagger,
+  message,
+  userComment,
+}) {
+  if (SKIP_FLAG_METHODS) {
+    return;
+  }
+  let titleLink;
+  let authorName;
+  let title = `Flag in ${flagger.profile.name}'s Inbox`;
+  let text = `${flagger.profile.name} (${flagger.id}; language: ${flagger.preferences.language}) flagged a PM`;
+  let footer = `<${SLACK_FLAGGING_FOOTER_LINK}?userId=${flagger.id}&chatMessageId=${message.id}|Flag this message>`;
+
+  if (userComment) {
+    text += ` and commented: ${userComment}`;
+  }
+
+  if (!message.user && message.uuid === 'system') {
+    authorName = 'System Message';
+  } else {
+    authorName = `${message.user} - ${authorEmail} - ${message.uuid}`;
+  }
+
+  let messageText = message.text;
+
+  if (flagger.id === message.uuid) {
+    messageText += `: ${flagger.profile.name} is writing to itself.`;
+  } else {
+    let sender = '';
+    let recipient = '';
+
+    if (message.sent) {
+      sender = flagger.profile.name;
+      recipient = message.user;
+    } else {
+      sender = message.user;
+      recipient = flagger.profile.name;
+    }
+
+    messageText += `: ${sender} is writing this message to ${recipient}.`;
+  }
+
+  flagSlack.send({
+    text,
+    attachments: [{
+      fallback: 'Flag Message',
+      color: 'danger',
+      author_name: authorName,
+      title,
+      title_link: titleLink,
```

Comment:
`title_link` can have the value "N/A" for private messages

Reply:
can the url be empty? so that it doesn't show it as an url ?

Reply:
Assuming the Slack side of the integration can handle that, sure! It's worth a try and if it breaks we can change it during testing.